recipes/mp-units/0.7.0/test_package/conanfile.py

```diff
@@ -1,25 +1,26 @@
-from conans import ConanFile, CMake, tools
-from conans.tools import Version
 import os
 
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, cmake_layout
+
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake_find_package_multi"
-    
-    # TODO remove when https://github.com/conan-io/conan/issues/7680 is solved (or VS2019 is updated to at least 16.9)
-    def _skip_check(self):
-        return self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) <= "16"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+
```

Comment:
```suggestion
    test_type = "explicit"

```

Reply:
I thought that `test_type` was just 1.x migration thing as described here: https://docs.conan.io/1/migrating_to_2.0/recipes.html?highlight=test_type#changes-in-the-test-package-recipe. Is it still mandatory in 2.0?

Reply:
Yes, looks like you're correct about that. I'm not used to reviewing libraries only meant for 2.0 yet.